### DISCUSSION TITLE
Error in SOLWEIG using POI

### DISCUSSION THREAD

**Discussion by sarahelisa94** [UMEP-dev/UMEP#348] [Q&A] [answered]
Hello,

For a master thesis I started to work with SOLWEIG. My goal is to validate measured data with the simulation data. I already ran the model without using a POI. After I added POI(s) I got the following error: 
'LINE 322 "poi_save[0, 10] = Kdown[int(poisxy[k, 2]), int(poisxy[k, 1])]"  ERROR MESSAGE: index 884937 is out of bounds for axis 0 with size 224'
I suppose that something is wrong with extent of  the POI grid. (This is also mentioned in the chapter footprint modelling in the UMEP tutorial). I changed the extent by adjusting it to the extent of my building and ground DSM. But I still get the error.

In the tutorial it is mentioned '_Measure the distance to the point of interest to the nearest boundary of the DSM data set.
Adjust the maximum fetch accordingly._'

Where can I adjust the maximum fetch?

Thanks for your help!

Kind regards,
Sarah

**Comment by biglimp** [comment, marked as answer]
Hi Sarah,
No, the fetch is only valid for the foot print plugin. If you have spotted this sentence in the SOLWEIG tutorial, please let us know where. 

Where you able to go through our own tutorial without errors (https://umep-docs.readthedocs.io/projects/tutorial/en/latest/Tutorials/IntroductionToSolweig.html#introtosolweig)? If yes, I suspect that you have a conflict in CRSs (coordinate reference systems) between your POI vector point file and your DSM.

**Reply by sarahelisa94** [reply]
Thanks a lot! It was indeed the wrong CRS. The run was successful now,
The sentence is not included in the SOLWEIG part. Since I was searching for an solution I just scanned the whole Tutorial document.

I wish you nice holidays!
